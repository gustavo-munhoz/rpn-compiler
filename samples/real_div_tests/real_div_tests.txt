(6 2 |)
(10 3 |)
(10 5 |)
(1 2 |)
(1 1 |)
(-10 2 |)
(10 -2 |)
(-10 -2 |)
(0 1 |)
(0 -1 |)
(12.5 1 |)
(12.5 -1 |)
(1000 0.01 |)
(1 0.000000059604644775390625 |)
(0.00006103515625 2 |)
(0.000000059604644775390625 2 |)
(80 15 |)