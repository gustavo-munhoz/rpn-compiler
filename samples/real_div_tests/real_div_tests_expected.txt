0x4200
0x42AA
0x4000
0x3800
0x3C00
0xC500
0xC500
0x4500
0x0000
0x8000
0x4A40
0xCA40
0x7C00
0x7C00
0x0000
0x0000
0x4555